modules/test/playwright/tests/frontend-data-set-views-web/tests/data-set-admin/selectionFilters.spec.ts

```diff
@@ -122,4 +123,88 @@ test.describe('Filters in Data Set Manager', () => {
 			).toBeVisible();
 		});
 	});
+
+	test('Preselected filter values are checked in the multiSelect', async ({
+		filtersPage,
+		page,
+		picklistApiHelpers,
+	}) => {
+		await test.step('Navigate to the Filters tab', async () => {
+			await filtersPage.goto({
```

Comment:
We can avoid the step as it's [part of `beforeEach()` ](https://github.com/liferay-frontend/liferay-portal/pull/4267/commits/c6703434da053c60bf5739ec8a17152142f2e683#diff-e86a8fda98500d2aea9059c1cf05bdc5d2428c4d19067b5f7f148d016defa609R59-R63)